src/components/faceVerification/screens/VerificationScreen.js

```diff
@@ -155,6 +155,7 @@ const FaceVerification = ({ screenProps }) => {
   const sdkExceptionHandler = useCallback(
     exception => {
       fireEvent(FV_ZOOMFAILED)
+      trackAttempt(exception)
```

Comment:
```suggestion
```

Reply:
if this somehow needed for stand alone fv flow (i don't really know why) - add `if (isFvFlow) {` condition
for generic in-app FV everything should be kept as is

Reply:
@roma-hladilka ^^^

Reply:
removed